src/routes/routes.ts

```diff
@@ -0,0 +1,45 @@
+import { Router } from "express";
+import {
+  editUserById,
+  getAllUsers,
+  getUserById,
+  createUser,
+  deleteUser,
+  createRegion,
+  deleteRegion,
+  editRegionById,
+  getRegionBySpecificPoint,
+  getRegionByDistance,
+} from "../controllers/index";
+
+export const routes = Router();
+
+/* ------------- USER ROUTES ------------- */
+
+// GET Routes
+routes.get("/users", getAllUsers);
+routes.get("/users/:id", getUserById);
+
+// POST Routes
+routes.post("/users", createUser);
+
+// PUT Routes
+routes.put("/users/:id", editUserById);
+
+// Delete Routes
+routes.delete("/users/:id", deleteUser);
+
+/* ------------- REGION ROUTES ------------- */
+
+// GET Routes
+routes.get("/regions/regionByPoint", getRegionBySpecificPoint);
+routes.get("/regions/regionByDistance", getRegionByDistance);
```

Comment:
gosto pessoal:
- usar kebab case pra rota
- não tem necessidade de repetir o nome da entidade, já vem do path

```suggestion
routes.get("/regions/by-point", getRegionBySpecificPoint);
routes.get("/regions/by-distance", getRegionByDistance);
```